app/src/features/reports/CompareMapNavReport.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState, useEffect } from "react";
+import { Grid } from "@material-ui/core";
+import UsaChloroplethMap from "../charts/UsaChloroplethMap";
+import TableChart from "../charts/TableChart";
+import WithDatasets from "../../utils/WithDatasets";
+import useDatasetStore from "../../utils/useDatasetStore";
+import variableProviders from "../../utils/variableProviders";
+import { Breakdowns } from "../../utils/Breakdowns";
+import VariableProvider from "../../utils/variables/VariableProvider";
+import { STATE_FIPS_MAP } from "../../utils/Fips";
+import Alert from "@material-ui/lab/Alert";
+import Breadcrumbs from "@material-ui/core/Breadcrumbs";
+import Link from "@material-ui/core/Link";
+import Typography from "@material-ui/core/Typography";
+
+interface Geo {
+  fips: number;
+  name: string;
+}
+
+function Map(props: {
+  fipsGeo: number;
+  data: Record<string, any>[];
+  updateGeoCallback: Function;
+}) {
+  function getStateGeoFromPropFips() {
+    return props.fipsGeo === 0
```

Comment:
What does 0/undefined represent? Can we make it required?

Reply:
the US string- with the other PR merged in, it's pointing to the USA_FIPS string now :)

Reply:
it was USA_FIPS. I removed this tho